### ISSUE TITLE
Missing Transform2D and Quat  constructors

### ISSUE THREAD

**Issue by qarmin** [godotengine/godot#32720]
**Godot version:**
3.2.alpha.custom_build .ed9fe4587

**OS/device including version:**
Ubuntu 19.04
**Issue description:**
In Godot two constructors which are present in docs not working correctly:
Quat constructor not working(I can't even run project)
```
var q_Quat : Quat = Quat(Vector3(randf(),randf(),randf()))
```
Transform2D always return null
```
var q_Transform2D = Transform2D(Transform( Vector3(randf(),randf(),randf()),Vector3(randf(),randf(),randf()),Vector3(randf(),randf(),randf()),Vector3(randf(),randf(),randf()) ))
q_Transform2D.affine_inverse()
```
Error - `Invalid call. Nonexistent function 'affinite_inverse' in base 'Nil'`

**Comment by aaronfranke** [comment]
Did these work previously?
What is the desired behavior of constructing a Transform2D from a Transform? For just copying values, to be in the same orientation, you'd need to flip Y. Should the constructor do this? 
https://github.com/godotengine/godot/blob/master/core/variant_call.cpp#L1923 doesn't seem to have any code for this operation.

**Comment by qarmin** [comment]
It is fixed in 4.0
